app/javascript/dashboard/routes/dashboard/settings/reports/components/CsatMetricCard.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="medium-2 csat--metric-card">
+    <h3 class="heading">
+      <span>{{ label }}</span>
+      <i v-tooltip="infoText" class="csat--icon ion-ios-information" />
+    </h3>
+    <h4 class="metric">
+      {{ value }}
+    </h4>
+  </div>
```

Comment:
Good candidate for story book as we have reports revamp coming in. This component would be noticed.

Reply:
Done.